frontend/src/app/shared/components/autocompleter/op-autocompleter/op-autocompleter.component.sass

```diff
@@ -1,50 +1,60 @@
 .op-autocompleter
-  &--option-wrapper
-    padding: 5px 5px
-    line-height: 15px
-    min-height: 25px  // line-height + padding
-    word-break: break-word
-    display: grid
-    grid-template-columns: auto 1fr
-    grid-template-rows: 1fr 1fr
+  &--option-wp
+    &-wrapper
+      padding: 5px 5px
+      line-height: 15px
+      min-height: 25px  // line-height + padding
+      word-break: break-word
+      display: grid
+      grid-template-columns: auto 1fr
+      grid-template-rows: 1fr 1fr
 
-  &--option-principal
-    margin-right: 5px
-    float: left
-    grid-column-start: 1
-    grid-column-end: 2
-    grid-row-start: 1
-    grid-row-end: 3
+    &-principal
+      margin-right: 5px
+      float: left
+      grid-column-start: 1
+      grid-column-end: 2
+      grid-row-start: 1
+      grid-row-end: 3
+
+    &-content
+      display: grid
+      grid-template-columns: auto 1fr auto
+      grid-template-areas: "project idlink status"
+      font-size: 0.8rem
+      padding: 1px 0 5px 0
 
-  &--wp-content
-    display: grid
-    grid-template-columns: auto 1fr auto
-    grid-template-areas: "project idlink status"
-    font-size: 0.8rem
-    padding: 1px 0 5px 0
+    &-project
+      grid-area: project
+      display: inline-block
+      overflow: hidden
+      white-space: nowrap
+      text-overflow: ellipsis
 
-  &--wp-project
-    grid-area: project
-    display: inline-block
-    overflow: hidden
-    white-space: nowrap
-    text-overflow: ellipsis
+    &-id
+      grid-area: idlink
+      place-self: right 
+      font-style: italic
+      margin: 0 0.5rem
+      text-align: center
 
-  &--wp-id
-    grid-area: idlink
-    place-self: right 
-    font-style: italic
-    margin: 0 0.5rem
-    text-align: center
+    &-status
+      grid-area: status
+      overflow: hidden
+      font-style: italic
 
-  &--wp-status
-    grid-area: status
-    overflow: hidden
-    font-style: italic
+    &-subject
+      font-weight: bold
+      display: inline-block
+      overflow: hidden
+      white-space: nowrap
+      text-overflow: ellipsis
 
-  &--wp-subject
-    font-weight: bold
-    display: inline-block
-    overflow: hidden
-    white-space: nowrap
-    text-overflow: ellipsis
\ No newline at end of file
+  &--option-principal
+    &-wrapper
+      display: flex
+      flex-direction: row
+    
+    &-avatar
+      margin-right: 5px
+      display:block
```

Comment:
That is invalid CSS due to the missing space. Since it works without that as well, it can be removed, too.